recipes-zarhus/packagegroups/packagegroup-zarhus.bb

```
@@ -24,3 +26,24 @@ RDEPENDS:${PN}-dbg = " \
     libgpiod-tools \
     devmem2 \
 "
+RDEPENDS:${PN}-alsa = " \
+    alsa-utils \
+    alsa-plugins \
+    alsa-lib \
+    alsa-tools \
+    alsa-state \
+    libasound \
+    alsa-utils-speakertest \
+    mpg123 \
+    gettext \
+    dialog \
+    ncurses \
+"
+
+RDEPENDS:${PN}-alsa-tools = " \
+    alsa-utils-speakertest \
+    mpg123 \
+    gettext \
+    dialog \
+    ncurses \
```

You can

```suggestion
    ${PN}-alsa \
```

if alsa-tools depends on alsa

---

Or maybe the other way include alsa-tools in alsa (though I think alsa-tools depends on alsa seems more correct)